src/loop_closure_assistant.cpp

```diff
@@ -199,20 +212,46 @@ void LoopClosureAssistant::publishGraph()
   edges_marker.color.a = 1;
   edges_marker.lifetime = rclcpp::Duration::from_seconds(0);
   edges_marker.points.reserve(edges.size() * 2);
+
+  visualization_msgs::msg::Marker localization_edges_marker;
+  localization_edges_marker.header.frame_id = map_frame_;
+  localization_edges_marker.header.stamp = node_->now();
+  localization_edges_marker.ns = "localization_edges";
```

Comment:
So this would change behavior for localization users. Is there a reason to not keep the initial namespace but just color them differently? Are you using this for anything more than visualization where this is an important detail? If not, using the same for other users for backward compatibility would be ideal

Reply:
Not a big deal.  I'll keep the namespaces as original.